frontend/test/__support__/server-mocks/search.ts

```diff
@@ -42,3 +44,26 @@ export function setupSearchEndpoints(
     options,
   );
 }
+
+export function setupEmbeddingDataPickerDecisionEndpoints(
+  dataPicker: EmbeddingDataPicker,
+) {
+  const mockEntityCount = match(dataPicker)
+    .with("flat", () => 10)
+    .with("staged", () => 100)
```

Comment:
This is the fallback mechanism for the embedding picker when users have 100 and more models and tables.